index.js

```diff
@@ -17,15 +19,8 @@ function Swarmerode () {
  * @param {Function} cb Callback with signature (err, hosts).
  */
 Swarmerode.prototype.swarmHosts = function (cb) {
-  this.swarmInfo(function (err, data) {
-    if (err) { return cb(err) }
-    var nodes = data.parsedSystemStatus.ParsedNodes
-    var hosts = Object.keys(nodes)
-      .map(function (key) {
-        return nodes[key].Host
-      })
-    cb(null, hosts)
-  })
+  var consul = new Consul()
+  consul.getSwarmNodes(cb)
```

Comment:
not sure this will fly everywhere. Some places (sauron) needs the org data from the host